lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToPercentiles.py

```diff
@@ -342,7 +350,7 @@ def test_spot_forecasts_check_data(self):
             "realization", iris.analysis.VARIANCE)
         percentiles = [10, 50, 90]
         plugin = Plugin()
-        result = plugin._mean_and_variance_to_percentiles(
+        result = plugin._location_and_scale_parameters_to_percentiles(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _location_and_scale_parameters_to_percentiles of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)